### DISCUSSION TITLE
recipes/sides/roasted-delicata-squash/

### DISCUSSION THREAD

**Discussion by giscus** [nicholaswilde/recipes#1179] [Comments]
# recipes/sides/roasted-delicata-squash/

A collection of my recipes.

https://nicholaswilde.io/recipes/sides/roasted-delicata-squash/

**Comment by nicholaswilde** [comment]
Sweet variation 
- 2 Tbsp maple syrup 
- 2 Tbsp olive oil 
- 1/2 tsp cinnamon
